documentation/docs/05-loading.md

```diff
@@ -93,65 +92,126 @@ If the page you're loading has an endpoint, the data returned from it is accessi
 
 `session` can be used to pass data from the server related to the current request, e.g. the current user. By default it is `undefined`. See [`getSession`](/docs/hooks#getsession) to learn how to use it.
 
-#### stuff
+#### parent
 
-`stuff` is passed from layouts to descendant layouts and pages, and can be filled with anything else you need to make available. For the root `__layout.svelte` component, it is equal to `{}`, but if that component's `load` function returns an object with a `stuff` property, it will be available to subsequent `load` functions.
+`parent` is passed from layouts to descendant layouts and pages, and contains a promise of the data they return. For the root `+layout.js` component, it is equal to `Promise.resolve({})`, but if that `load` function returns data, it will be available to subsequent `load` functions.
 
-#### status
+```js
+/// file: src/routes/cart/+page.js
+/** @type {import('./$types').Load} */
+export async function load({ parent }) {
+	const user = await parent();
 
-`status` is the HTTP status code when rendering an error page, or `null` otherwise.
+	return {
+		items: await getCartItems(user.cart);
+	};
+}
+```
 
-#### error
+#### depends
 
-`error` is the error that was thrown (or returned from a previous `load`) when rendering an error page, or `null` otherwise.
+`depends` is a helper method which allows to declare a dependency on a specific resource. The input is an array of strings representing URLs the page depends on, which can subsequently be used with [`invalidate`](/docs/modules#$app-navigation-invalidate) to cause `load` to rerun. You only need to invoke `depends` with them if you're using a custom API client; URLs loaded with the provided `fetch` function are added automatically.
```

Comment:
Text say "The input is an array of strings..." but in example it's not array of strings but just string. Confusing.

Reply:
Good catch